package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "arlas-wui-toolkit",
-  "version": "12.5.3",
+  "name": "@gisaia-team/arlas-wui-toolkit",
+  "version": "12.6.0-dev0",
```

Comment:
Should we use snapshot?

Reply:
This issue is global to all projects and should be fixed in arlas-web-core